lib/raven/processor/sanitizedata.rb

```diff
@@ -28,6 +30,13 @@ def sanitize(k,v)
 
     private
 
+    def sanitize_query_string(query_string)
+      query_string.split('&').map do |key_val_pairs|
```

Comment:
Can you kindly use `CGI::parse` here? I think the code as written will break with keys that have several values + url encoding.